docs/external/finops.md

```
@@ -0,0 +1,55 @@
+```json
+{
+  "sub": "<EXT_FFC_SUB>",
+  "exp": <now + 5 minutes>,
+  "nbf": <now>,
+  "iat": <now>
+}
+```
+
+Token is signed with `HS256` using `EXT_FFC_OPERATIONS_SECRET` and attached via:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)